dbms/src/IO/createReadBufferFromFileBase.cpp

```diff
@@ -23,8 +25,12 @@ std::unique_ptr<ReadBufferFromFileBase> createReadBufferFromFileBase(const std::
 	}
 	else
 	{
+#ifndef __APPLE__
```

Comment:
Also better to ignore aio_threshold. Otherwise server will throw exceptions in if we activate aio_threshold in default config. So, aio_threshold setting must be ignored on Mac.